dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -1012,7 +1012,7 @@ std::vector<BlobFileId> BlobStore::getGCStats()
             if (right_margin != stat->sm_total_size)
             {
                 auto blobfile = getBlobFile(stat->id);
-                LOG_FMT_TRACE(log, "Truncate blob file [blob_id={}] [origin size={}] [truncated size={}]", stat->id, stat->sm_total_size, right_margin);
+                LOG_FMT_INFO(log, "Truncate blob file [blob_id={}] [origin size={}] [truncated size={}]", stat->id, stat->sm_total_size, right_margin);
```

Comment:
keep this log level to be "trace", and append the right margin into `blobstore_gc_info.appendToTruncatedBlob` and log within one line?

Reply:
```suggestion
                LOG_FMT_TRACE(log, "Truncate blob file [blob_id={}] [origin size={}] [truncated size={}]", stat->id, stat->sm_total_size, right_margin);
```

I think this line should keep as trace level, since we will print the info by `blobstore_gc_info.toString()` in L1051

Reply:
too many logs may make logs rotation happen too often, and some logs be thrown away under Cloud service